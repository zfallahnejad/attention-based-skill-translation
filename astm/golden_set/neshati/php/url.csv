548225,url
851273,url
138475,url
1338292,url
50079,url
127724,url
20862,url
889949,url
1438393,url
42139,url
