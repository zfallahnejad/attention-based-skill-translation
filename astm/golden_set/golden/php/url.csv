548225,42,37,0.5316455696202531
851273,24,15,0.6153846153846154
138475,10,7,0.5882352941176471
50079,9,5,0.6428571428571429
1338292,9,11,0.45
127724,8,5,0.6153846153846154
20862,7,10,0.4117647058823529
42139,7,7,0.5
889949,7,6,0.5384615384615384
1438393,7,3,0.7
